pkg/ccl/backupccl/restore_online.go

```diff
@@ -108,15 +113,51 @@ func sendAddRemoteSSTs(
 
 func sendAddRemoteSSTWorker(
 	execCtx sql.JobExecContext,
+	tableRekeys []execinfrapb.TableRekey,
+	tenantRekeys []execinfrapb.TenantRekey,
+	elide execinfrapb.ElidePrefix,
+	fromSystemTenant bool,
 	restoreSpanEntriesCh <-chan execinfrapb.RestoreSpanEntry,
 	requestFinishedCh chan<- struct{},
-	fromSystemTenant bool,
 ) func(context.Context) error {
 	return func(ctx context.Context) error {
 		var toAdd []execinfrapb.RestoreFileSpec
 		var batchSize int64
 		const targetBatchSize = 440 << 20
 
+		// TODO(dt): handle tenants
```

Comment:
There is a tenant rekey bit below, so mind putting a few more words here about what still needs to be handled?